README.md

```diff
@@ -72,29 +78,41 @@ curl -s -G --data-urlencode "q=title:list" localhost:9200/try_cloaked_search/_se
 These are a couple examples of simple term queries:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:Japan" localhost:8675/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND title:Japan" localhost:8675/try_cloaked_search/_search | jq
```

Comment:
Ditto on the quoting. This used to be querying a protected field, which showed that the proxy was doing its job. This is now an unprotected field. I see you expanded below to indicate that we change for querying unprotected fields to protected ones, but that isn't clear at this point in the document.